health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -348,6 +424,234 @@ function UploadDataMapping({ formData, onSelect, currentCategories }) {
+          if (property?.isRequired && required.indexOf(property?.name) === -1) {
+            required.push(property?.name);
+          }
+
+          // If orderNumber is missing, default to a very high number
+          columns.push({ name: property?.name, orderNumber: property?.orderNumber || 9999999999 });
+        }
+      }
+    }
+    enrichSchema(convertData, properties, required, columns);
+    const newData = JSON.parse(JSON.stringify(convertData));
+    delete newData.campaignType;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)